debian/control

```diff
@@ -251,6 +251,7 @@ Depends: ceph-mgr (= ${binary:Version}),
          ${misc:Depends},
          ${python:Depends},
          ${shlibs:Depends},
+Suggests: python3-saml
```

Comment:
i cannot find python3-saml in either debian or ubuntu official repo:

- https://packages.debian.org/search?keywords=python3-saml&searchon=names&suite=all&section=all
- https://packages.ubuntu.com/search?keywords=python3-saml&searchon=names&suite=all&section=all

Reply:
Yup, I know it, I checked it yesterday :/ As it's a weak dependency, it shouldn't block the install, right?

Reply:
i'd prefer drop this line. if we cannot offer this package.

Reply:
i am copying the discussion over IRC here

----

[20:38:23]  <epuertat__>	kefu, I prefer to 'track' that dependency in the package metadata. being a weak dependency it shouldn't block the installation if it's not found, and weak deps were created exactly for specifying optional components. IMHO it doesn't make sense to define it in the requirements.txt (where it's not optional) but not in the package spec
[20:39:29]  <kefu>	epuertat__ i think the reason we list a package in packaging is to help maintainer and enduser to consumer this software.
[20:41:24]  <kefu>	if we cannot offer them the package, and we don't plan to do so, why shall we list it in, for instance, d/control?
[20:43:11]  <kefu>	FWIW, podman is being ITP'ed.
[20:43:45]  <epuertat__>	kefu, I get your point, but if we follow that reasoning why we added python3-saml dependencies (xmlsec, ...) to the BuildRequires?
[20:44:16]  <kefu>	epuertat__ strictly speaking, they are not there.
[20:44:56]  <epuertat__>	kefu, by specifying the weak deps, we allow users to build those deps on their (or in the future if someone comes up with those)
[20:45:22]  <kefu>	because we use builddeps track build, make check, and runtime deps.
[20:46:36]  <kefu>	instead of scattering them around, i'd prefer put all of them in a single place. and guard them with either %if macro (in rpm spec) or comments in d/control.
[20:47:06]  <kefu>	i don't agree with "allow".
[20:47:33]  <epuertat__>	kefu, don't you mind if we follow this discussion in the PR, so we track any decision there?
[20:47:46]  <kefu>	if we want to help our user. a README or a doc is better.

Reply:
Thanks. No problem, I will remove this line.

@ricardoasmarques , do you (SUSE) want this dependency also removed from ceph.spec?